ingestion/src/metadata/profiler/profiler/sampler.py

```diff
@@ -40,6 +42,27 @@
 RANDOM_LABEL = "random"
 
 
+def _object_value_for_elem(self, elem):
```

Comment:
Override this function, Becouse in Enum it was giving KeyError from here `return self._object_lookup[elem]`.
now make this to` return self._object_lookup.get(elem, elem)`.

Reply:
Not sure if returning `elem` if this fails when getting the values from the dict is the right approach or we would be altering the reality. Thinking if returning `None` might be better suited for this.

@TeddyCr what are your thoughts on this?

Reply:
@pmbrull so [here](https://github.com/open-metadata/OpenMetadata/blob/f54efc649b4bcbf154ba78c53ca23787b9aee671/ingestion/src/metadata/profiler/orm/converter.py#L66) we have mapped enum type to sqlalchemy.Enum and we are supposed to pass the enum associated with this column (checkout the first code snippet) now we do not store this enum map which is required 

![image](https://user-images.githubusercontent.com/39544459/227856199-d2917ff4-3019-40fd-8c5d-d4564b88f4b9.png)


So in that case, if we map by default return None, we will always get None because there is no enum map to lookup, so what we are doing here is basically trusting the database that it will be storing the correct map key and showing directly that on the UI, but even in this approach we will be only able to display what database has stored (i.e the key) and not the actual value of the same! 

![image](https://user-images.githubusercontent.com/39544459/227856427-015f6765-6e38-4eaf-bc0f-a80fd7aa9cf4.png)